packages/tree-select/package.json

```diff
@@ -0,0 +1,37 @@
+{
+	"name": "@automattic/tree-select",
+	"version": "0.1.0",
+	"description": "A selector library for Redux",
+	"keywords": [
+		"redux",
+		"selector"
+	],
+	"author": "Ben Lowery <ben.lowery@automattic.com>",
+	"homepage": "https://github.com/Automattic/wp-calypso",
+	"license": "GPL-2.0-only",
+	"main": "dist/cjs/index.js",
+	"module": "dist/esm/index.js",
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/Automattic/wp-calypso.git"
+	},
+	"publishConfig": {
+		"access": "public"
+	},
+	"scripts": {
+		"prepare": "npm run build",
+		"build": "node ../../bin/build-package.js",
+		"clean": "node ../../bin/clean-package.js"
+	},
+	"bugs": {
+		"url": "https://github.com/Automattic/wp-calypso/issues"
+	},
+	"files": [
+		"dist",
+		"src"
+	],
+	"dependencies": {
+		"lodash": "^4.17.0"
+	},
+	"gitHead": "5aeb53f1c33fff67a7c1773f2d95e79e8d0a91b1"
```

Comment:
The `gitHead` property should only be in the published `package.json`. (`lerna publish` adds it there, in addition to other modifications like changing `file:` links to actual version numbers.) Shouldn't be a part of the `package.json` committed to Git.

Reply:
That's... fun.

Reply:
hmm... seems like [the default on `--git-reset`](https://github.com/lerna/lerna/tree/master/commands/publish#--git-reset) should have undone this for me. Wonder what happened.